### DISCUSSION TITLE
Thrustmaster idiosyncracies or my wn fault?

### DISCUSSION THREAD

**Discussion by Phantomflyer81** [flybywiresim/aircraft#4842] [Q&A]
Hello folks,

I set up my TCA , Stick and TFRP rudder (All Thrustmaster for A320). I installed the latest experimental mod (6.1 stable didn't allow me to calibrate and leave a ThrottleConfiguration ini in the "work" folder (created automatically??)). 
So I used the preset ini from other users here as a starting point. Then I did a fine calibration with the  Fl;yPad . It saved the new config no problem and the detents work quite well, as do the rudder pedals and side stick. The side-stick has its own throttle, but I obviously don't need (or want) to use it. If this might be the reason for the throttle advancing to TOGA upon view change (external to cockpit), how would I go about disabling it? Can I do that in sim or using the FlyPad? 

The problem I'm having is upon switching views from external to cockpit, the throttles spool up to TOGA position. It keeps me on my] toes (brakes :)) alright, and I need to advance the throttles then retard to stop this behaviour. 

Another problem (not critical) I'm encountering is no ability to map a button to retract the spoilers after landing and turning off hte runway. I need to do it from the cockpit itself.

Oh, I forgot to ask what the .fdr (Flight Data Recorder) file is that is created in the work folder?

Here's my Throttle config numbers in case anyone wants to view them. 

[Throttle]
Log = true
Enabled = true
ReverseOnAxis = true
ReverseIdle = true
DetentDeadZone = 2.0
DetentReverseFull = -1.00
DetentReverseIdle = -0.69
DetentIdle = -0.50
DetentClimb = -0.02
DetentFlexMct = 0.49
DetentTakeOffGoAround = 1.00

[throttle_common]
reverse_on_axis = true

[throttle_axis_1]
reverse_low = -0.700000
reverse_high = -1.000000
reverse_idle_low = -0.560000
reverse_idle_high = -0.690000
idle_low = -0.550000
idle_high = -0.470000
climb_low = -0.020000
climb_high = 0.020000
flex_mct_low = 0.470000
flex_mct_high = 0.510000
toga_low = 0.980000
toga_high = 1.000000

[throttle_axis_2]
reverse_low = -0.700000
reverse_high = -1.000000
reverse_idle_low = -0.560000
reverse_idle_high = -0.690000
idle_low = -0.550000
idle_high = -0.470000
climb_low = -0.020000
climb_high = 0.020000
flex_mct_low = 0.470000
flex_mct_high = 0.510000
toga_low = 0.980000
toga_high = 1.000000

Thanks in advance!

**Comment by ExampleWasTaken** [comment]
The experimental version is and was never supported. Apart from that, please use the development version as the experimental version is not updated at the moment. 
The FDR file is indeed the flight data recorder file used to record various parameters during the flight.

**Reply by Phantomflyer81** [reply]
Someone should tell the real airbus pilot (UK) on YouTube about not using the experimental version because it's the only one he says works with Thrustmaster! I guess I will refer him here.

**Reply by tracernz** [reply]
> Someone should tell the real airbus pilot (UK) on YouTube about not using the experimental version because it's the only one he says works with Thrustmaster! I guess I will refer him here.

Keep in mind this all changes over time and many videos were recorded in the past when experimental may have been in use.

**Reply by Phantomflyer81** [reply]
SO which version should I use? Dev or 6.1 stable? And if I do, will I need to go through all the calibration again?

**Reply by tracernz** [reply]
> SO which version should I use? Dev or 6.1 stable? And if I do, will I need to go through all the calibration again?

If you want the latest and greatest, including custom autopilot + FBW, and throttle calibration on the EFB, then you need to run dev (master).

**Reply by Phantomflyer81** [reply]
I have the installer and it doesn't show "master" - only development.

**Reply by ExampleWasTaken** [reply]
> 
> 
> I have the installer and it doesn't show "master" - only development.

"development" is the production name of the master branch

**Reply by Phantomflyer81** [reply]
Thanks folks. The dev master version definitely better all round. Really enjoying the FBW using this Airbus gear. Calibrating on the FlyPad was great.

**Comment by Riggie** [comment]
" The problem I'm having is upon switching views from external to cockpit, the throttles spool up to TOGA position. "   I have this problem also, really annoying, within seconds the A320xn dev version is at 370kn, the button I use to change views is on the side stick and is not mapped to any other operation. 

" I'm encountering is no ability to map a button to retract the spoilers after landing " 
 I  struggle with the retracting also,  I found pressing the button for just a slightly longer time and it does work ........... it is mapped,  button 4 "Toggle Spoilers"  on the Airbus sidestick and it has worked OK for months now.

**Reply by Phantomflyer81** [reply]
Yes and thanks for the tip on retraction of spoilers after landing. Hold the button down a little longer. But the switch view and instant
100% power not good!! I get ready for it before switching back.
